easybuild/easyconfigs/l/LAMMPS/LAMMPS-23Jun2022-foss-2021a-CUDA-kokkos.eb

```
@@ -0,0 +1,166 @@
+name = 'LAMMPS'
+version = '23Jun2022'
+versionsuffix = '-CUDA'
+versionsuffix += '-kokkos'
```

as far as I'm aware, all easyconfigs include the CUDA version in the suffix
```suggestion
_cuda_suffix = '-CUDA-%(cudaver)s'
versionsuffix = _cuda_suffix + '-kokkos'
```